helpers/for.js

```diff
@@ -3,29 +3,30 @@
 const _ = require('lodash');
 
 const factory = () => {
-    return function(from, to, context) {
-        const options = arguments[arguments.length - 1];
+    return function(from, to, context, options) {
         const maxIterations = 100;
-        var output = '';
+        let output = '';
 
         function isOptions(obj) {
             return _.isObject(obj) && obj.fn;
         }
 
         if (isOptions(to)) {
+            options = to;
             context = {};
             to = from;
             from = 1;
 
         } else if (isOptions(context)) {
+            options = context;
             if (_.isObject(to)) {
                 context = to;
                 to = from;
                 from = 1;
             }
         }
 
-        if (to <= from) {
+        if (to < from) {
```

Comment:
Why was this changed? Seems like this will alter current looping logic.

Reply:
It allows for one iteration.  Previously we could only do two iterations at the minimum.  There was an issue written about it too.

Reply:
Let's pull this out as a separate change so we can evaluate it separately.